packages/analytics/dashboard-renderer/package.json

```diff
@@ -79,6 +79,7 @@
   },
   "dependencies": {
     "@kong-ui-public/core": "workspace:^",
+    "@kong-ui-public/entities-shared": "workspace:^",
```

Comment:
What does this dependency do?

Reply:
Need it to use <EntityLink>

Reply:
`entities-shared` is massive: 
![image](https://github.com/Kong/public-ui-components/assets/55025/c918c73b-60a0-48d4-89cf-77189d000c28)

Can you please make it a peer dependency, externalize it in `vite.config.ts`, and add it to the peer dependency requirements listed in the README?  Hopefully that lets you bump the `distSizeChecker` limit back down too.

Reply:
So what's the verdict, should this be taken care of in this PR or part of a change to entities-shared like @adamdehaven suggested?